akka-cluster-typed/src/main/scala/akka/cluster/ddata/typed/javadsl/Replicator.scala

```diff
@@ -296,13 +318,8 @@ object Replicator {
     def key: Key[A]
   }
   final case class DeleteSuccess[A <: ReplicatedData](key: Key[A]) extends DeleteResponse[A]
-  final case class ReplicationDeleteFailure[A <: ReplicatedData](key: Key[A]) extends DeleteResponse[A]
-  final case class DataDeleted[A <: ReplicatedData](key: Key[A])
-      extends RuntimeException
-      with NoStackTrace
-      with DeleteResponse[A] {
-    override def toString: String = s"DataDeleted [$key]"
-  }
+  final case class DeleteFailure[A <: ReplicatedData](key: Key[A]) extends DeleteResponse[A]
+  final case class DataDeleted[A <: ReplicatedData](key: Key[A]) extends DeleteResponse[A]
```

Comment:
If we tried to delete and it already was deleted, isn't that a success?

Reply:
That would be one way to define it and then this `DataDeleted` wouldn't be needed. Now it's not like that, and it makes the distinction between the two cases. Maybe leave that as is rather than changing more behavior?

Reply:
Alright, lets' stay with current then.